pkg/webhooks/pod_webhook.go

```diff
@@ -113,30 +120,65 @@ func (p *PodWebhook) Default(ctx context.Context, obj runtime.Object) error {
 		} else {
 			groupUniqueKey = pod.Labels[leaderworkerset.GroupUniqueHashLabelKey]
 		}
-		_, foundEpKey := pod.Annotations[leaderworkerset.ExclusiveKeyAnnotationKey]
-		if foundEpKey && !exclusiveAffinityApplied(*pod) {
-			SetExclusiveAffinities(pod, groupUniqueKey)
+		epKey, foundEpKey := pod.Annotations[leaderworkerset.ExclusiveKeyAnnotationKey]
+		if (foundEpKey) && !exclusiveAffinityApplied(*pod, epKey) {
+			SetExclusiveAffinities(pod, groupUniqueKey, epKey, leaderworkerset.GroupUniqueHashLabelKey)
+		}
```

Comment:
```suggestion
		
		if epKey, foundEpKey := pod.Annotations[leaderworkerset.ExclusiveKeyAnnotationKey]; foundEpKey {
			SetExclusiveAffinities(pod, groupUniqueKey, epKey, leaderworkerset.GroupUniqueHashLabelKey)
		}
```

and call `exclusiveAffinityApplied(*pod, epKey)` inside `SetExclusiveAffinities`